Formula/xgboost.rb

```diff
@@ -0,0 +1,26 @@
+class Xgboost < Formula
+  desc "Scalable, Portable and Distributed Gradient Boosting Library"
+  homepage "https://xgboost.ai/"
+  url "https://github.com/dmlc/xgboost.git",
```

Comment:
Is there a reason that the git repo is used as opposed to the tarball which is the preferred option?

Reply:
Yeah, the repo has submodules. I created an earlier version with resources (link below), but this seemed cleaner/more maintainable.

https://github.com/Homebrew/homebrew-core/compare/master...ankane:xgboost1